### DISCUSSION TITLE
Adding Animations to the Signup page (Both student and admin) as well as other parts of the project

### DISCUSSION THREAD

**Discussion by krishbedi17** [prathamgarg03/Library-Management-System-CMPT-276-#3] [Ideas]
I have sent everyone some links to beginner guides and other helpful resources on how to make the website a bit more interactive. Please check them out and let me know if we everyone is willing to implement them into our project.

If you have not had a chance to look at the message, I have attached the links with this forum as well. 

Links:

Guides to help get started:
CSS Animations and Transitions:

CSS Animation for Beginners (https://thoughtbot.com/blog/css-animation-for-beginners)
A Comprehensive Guide to CSS Transitions (https://www.smashingmagazine.com/2020/04/comprehensive-guide-css-transitions/)
CSS Animation Tutorial for Beginners | Dev Ed (https://www.youtube.com/watch?v=LB9dCZ2Zg3E)

JavaScript Animation Libraries:
GSAP Getting Started (https://greensock.com/get-started/)
Anime.js Documentation (https://animejs.com/documentation/)
JavaScript Animation Tutorial | The Net Ninja (https://netninja.dev/p/javascript-animation-tutorial)

SVG Animations:
Getting Started with Snap.svg (http://snapsvg.io/start/)
SVG Animation Using CSS and Snap.svg | CSS-Tricks(https://css-tricks.com/scale-svg/)
Snap.svg Animation Tutorial | Traversy Media (https://www.youtube.com/watch?v=le-72sUbDmc)

Other resources:

CSS Animations and Transitions:
CSS Transitions (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Transitions)
CSS Animations (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations)
Animate.css (https://animate.style/)

JavaScript Animation Libraries:
GreenSock Animation Platform (GSAP) (https://greensock.com/gsap/)

SVG Animations:
Snap.svg (http://snapsvg.io/)

Web Animation API:
Web Animation API (https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API)

CSS Frameworks:
Bootstrap (https://getbootstrap.com/)

JavaScript Libraries for Scroll Animations:
ScrollMagic (https://scrollmagic.io/)
AOS (Animate On Scroll) (https://michalsnik.github.io/aos/)

CSS Animation Generators:
Animista (https://animista.net/)
Keyframes App (https://keyframes.app/)
Animate It(https://animate.style/)

**Comment by Gourav-Sharma243** [comment]
Yes and you can even use the link that steve gave . I mean the playlist of boby chan let me know if i should pin the link in this form
